packages/dropdowns.next/src/views/menu/StyledMenu.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  retrieveComponentStyles,
+  DEFAULT_THEME,
+  arrowStyles,
+  ARROW_POSITION as ArrowPosition
+} from '@zendeskgarden/react-theming';
+import { IStyledListboxProps, StyledListbox } from '../combobox/StyledListbox';
+
+const COMPONENT_ID = 'dropdowns.menu';
+
+interface IStyledMenuProps extends IStyledListboxProps {
+  arrowPosition?: ArrowPosition;
+}
+
+/*
+ * 1. Override arrow parent positioning to ensure arrow is visible beyond block overflow boundaries.
+ */
+export const StyledMenu = styled(StyledListbox).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledMenuProps>`
+  /* stylelint-disable-next-line declaration-no-important */
+  position: static !important; /* [1] */
```

Comment:
Are we expecting the menu to have a scrollbar gutter visible at all times? If not, we could set `overflow-y` to `auto`.

Current:
![menu-with-scrollbar-gutter](https://github.com/zendeskgarden/react-components/assets/12474067/59685ef3-2906-40ae-ae6c-ced3f47cb0bb)
Without scrollbar gutter:
![menu-without-scrollbar-gutter](https://github.com/zendeskgarden/react-components/assets/12474067/5e299859-ab89-4759-b695-d5f93e4c1dc7)

Reply:
Ack, we never want that – thanks for the catch.